linux_os/guide/services/ssh/ssh_server/sshd_limit_access/rule.yml

```diff
@@ -0,0 +1,47 @@
+documentation_complete: true
+
+prodtype: ubuntu2004,ubuntu2204
+
+title: 'Restrict sshd User Access via AllowUsers, AllowGroups, DenyUsers or DenyGroups'
+
+description: |-
+    AllowUsers gives the system administrator the option of allowing
+    specific users to ssh into the system.
+    - The list consists of space separated user names
+    - Numeric user IDs are not recognized with this variable
+    - A system administrator may restrict user access further by only
+      allowing the allowed users to log in from a particular host by
+      specifying the entry as user@host.
+    AllowGroups gives the system administrator the option of allowing
+    specific groups to ssh into the system.
+    - The list consists of space separated group names
+    - Numeric group IDs are not recognized with this variable
+    DenyUsers gives the system administrator the option of denying
+    specific users to ssh into the system.
+    - The list consists of space separated user names
+    - Numeric user IDs are not recognized with this variable
+    - A system administrator may restrict user access further by only
+      allowing the allowed users to log in from a particular host by
+      specifying the entry as user@host.
```

Comment:
This is the very same comment as for `AllowUsers`.
I would make the comment relevant for the `DenyUsers` option. Something like:
```
A system administrator may restrict user access with greater granularity by
denying users log in from a particular host specifying the entry as user@host
```